ros_buildfarm/sourcerpm_job.py

```diff
@@ -20,7 +20,7 @@
 
 def _get_source_tag(
         rosdistro_name, pkg_name, pkg_version, os_name, os_code_name):
-    assert os_name in ['centos', 'fedora', 'rhel']
+    assert os_name in ['fedora', 'rhel']
```

Comment:
My initial thought was that if Fedora is supported here it is conceivable that future CentOS in its "Stream" role may also be supported here. But if that's ever the case we can add something back for it.

Reply:
That's an interesting thought - that CentOS Stream isn't a RHEL clone like CentOS itself is. Let's punt that for now - I think other infrastructure like `rosdep`/`rosdistro` might need to be updated to make that work anyway.